### DISCUSSION TITLE
Editing shortcuts directly from Most Visited Sites feature

### DISCUSSION THREAD

**Discussion by vanshsinhaa** [dailydotdev/daily#1671] [Feature Requests]
I noticed an issue with the shortcuts menu that could be improved. There are two tabs: "My Shortcuts" and "Most Visited Sites". The problem is with the "Most Visited Sites" tab. If I want to replace just one website from this list, I can’t do it directly. Instead, I have to copy all the links from the "Most Visited Sites" tab, manually add them to the "My Shortcuts" tab, and then include the new website I want.

This process feels unnecessarily complicated. It would be much more user-friendly if we could edit or replace individual websites directly within the "Most Visited Sites" tab. That way, I wouldn’t have to go through the extra steps of manually recreating the entire list in "My Shortcuts" just to make one small change.

**Comment by idoshamun** [comment]
Hi @VanshSinha18,

I understand the issue you're experiencing.
To give you context for this behavior, when selecting most visited sites we fetch the list from Chrome, my shortcuts is fetched from our server. We can make the transition seamless, as you suggest, but I just wanted to explain how it works.

**Reply by vanshsinhaa** [reply]
Of course, I understand that. I was just giving you suggestion to make your software more user friendly. I appreciate your explanation and appreciate your response. Look forward to seeing your website prosper.
